src/main/scala/com.snowplowanalytics.maxmind.iplookups/IpLookups.scala

```diff
@@ -190,7 +195,11 @@ class IpLookups(
       result
   }
 
-  /** Transforms a String into an Validation[Throwable, InetAddress] */
-  private def getIpAddress(ip: String): Validation[Throwable, InetAddress] =
-    Validation.fromTryCatch(InetAddress.getByName(ip))
+  /** Transforms a String into an Either[IpLookupError, InetAddress] */
+  private def getIpAddress(ip: String): Either[IpLookupError, InetAddress] =
+    try {
+      Right(InetAddress.getByName(ip))
+    } catch {
+      case ex: java.net.UnknownHostException => Left(UnknownHostException(ex.getMessage))
+    }
```

Comment:
you can just do [Either.catchNonFatal(...)](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/syntax/either.scala#L306)